apps/api/docs/channels/sms-kapsystem.md

```diff
@@ -0,0 +1,49 @@
+# KAPSystem SMS
+
+KAPSystem enables sending SMS through its API, allowing developers to programmatically send and receive text messages worldwide. Users integrate KAPSystem's API into their applications, triggering automated SMS messages for various purposes such as notifications, alerts, and customer communications.
+
+Refer to the messaging documentation to dive deeper into the features Plivo offers and understand how it all works.
```

Comment:
Check and ensure "Plivo" is not mentioned in this documentation.

Reply:
Fixed